configs/world.conf

```diff
@@ -510,7 +510,7 @@
 #        Default: 0
 #
 
-<FloodProtection Lines = "0" Seconds = "0" SendMessage = "0">
+                 SendMessage = "0">
```

Comment:
You missed some values
Old: <FloodProtection Lines = "0" Seconds = "0" SendMessage = "0">